polyhost/util/crash_log.py

```
@@ -0,0 +1,170 @@
+    def _thread_hook(args):
+        if args.exc_type is SystemExit:
+            return
+        try:
+            log.critical("UNHANDLED EXCEPTION in thread %r",
+                         getattr(args.thread, "name", "?"),
+                         exc_info=(args.exc_type, args.exc_value, args.exc_traceback))
+            _flush_logging()
+            _stamp("thread exception (%s): %s"
+                   % (getattr(args.thread, "name", "?"), args.exc_value))
+        except Exception:  # noqa: BLE001
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)